Sitemap/SitemapAwareDocumentVoter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+use Symfony\Cmf\Bundle\SeoBundle\SitemapAwareInterface;
+
+/**
+ * Implementing the SitemapAwareInterface will give the chance the let the document decide
+ * whether it wants to be displayed on sitemap or not.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class SitemapAwareDocumentVoter implements VoterInterface
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function exposeOnSitemap($content, $sitemap)
+    {
+        return $content instanceof SitemapAwareInterface && $content->isVisibleInSitemap();
```

Comment:
the voters need to be unanimous, right? and there is no 3rd option "abstain"? i think whe need to say `if (!$content instanceof SitemapAwareInterface) {return true;}`